src/VirtualClient/VirtualClient.Actions.UnitTests/Examples/SPECview/resultCSV_multipleViewSets.csv

```diff
@@ -0,0 +1,89 @@
+Composites
+3dsmax-07,35.46
+catia-06,43.3
+creo-03,56.98
+energy-03,27.46
+maya-06,227.8
+medical-03,35.16
+snx-04,139.11
+solidworks-07,97.8
+
+viewset,index,name,weight,fps
+3dsmax-07,1,3dsmax_Arch_Shaded,9.52,50.87
```

Comment:
I think it is valuable to parse out all the sub tests instead of just the composites. What do you think? It will help us debug why the composite goes lower, and help us find why somethign changes

Reply:
Will add this to the next PR, thanks!